### PR TITLE
MULE-8455 Upgrade license checker

### PR DESCRIPTION
**Pull request by vromero**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1697,10 +1696,14 @@
                 <plugin>
                     <groupId>com.mycila</groupId>
                     <artifactId>license-maven-plugin</artifactId>
-                    <version>2.6</version>
+                    <version>2.8</version>
                     <configuration>
                         <skip>${skipVerifications}</skip>
-                        <header>${licensePath}</header>
+                        <header>com/mycila/maven/plugin/license/templates/CPAL.txt</header>
```
Comment (marianogonzalez) [inline pom.xml]: are you willing to bet the little hair you have left that this generates the exact same header as we have now?

Comment (marianogonzalez) [inline pom.xml]: how about EE? Do you have to configure the plugin again? you will need a file there because that license is custom

Comment (vromero) [inline pom.xml]: I can bet it is the same file as I personally did the pull request to the mycila project. And of course I tested locally before commiting.
For EE it has to remain as it is, the license is not included (obviously) in the license checker distribution.

Comment (marianogonzalez) [inline pom.xml]: what do you mean that EE has to remain as it is? EE inherits the plugin definition from CE, so at the very least you have to override it

Comment (vromero) [inline pom.xml]: you are right about EE. I've created a parallel PR https://github.com/mulesoft/mule-ee/pull/733

**Pull request merged**